src/components/TaskHeader.js

```diff
@@ -0,0 +1,120 @@
+import React, {useEffect} from 'react';
+import {View, TouchableOpacity} from 'react-native';
+import PropTypes from 'prop-types';
+import lodashGet from 'lodash/get';
+import reportPropTypes from '../pages/reportPropTypes';
+import withLocalize, {withLocalizePropTypes} from './withLocalize';
+import * as ReportUtils from '../libs/ReportUtils';
+import * as Expensicons from './Icon/Expensicons';
+import Text from './Text';
+import participantPropTypes from './participantPropTypes';
+import Avatar from './Avatar';
+import styles from '../styles/styles';
+import themeColors from '../styles/themes/default';
+import CONST from '../CONST';
+import withWindowDimensions from './withWindowDimensions';
+import compose from '../libs/compose';
+import Navigation from '../libs/Navigation/Navigation';
+import ROUTES from '../ROUTES';
+import Icon from './Icon';
+import MenuItemWithTopDescription from './MenuItemWithTopDescription';
+import Button from './Button';
+import * as TaskUtils from '../libs/actions/Task';
+
+const propTypes = {
+    /** The report currently being looked at */
+    report: reportPropTypes.isRequired,
+
+    /** Personal details so we can get the ones for the report participants */
+    personalDetails: PropTypes.objectOf(participantPropTypes).isRequired,
+
+    ...withLocalizePropTypes,
+};
+
+function TaskHeader(props) {
+    const title = ReportUtils.getReportName(props.report);
+    const assigneeName = ReportUtils.getDisplayNameForParticipant(props.report.managerEmail);
+    const assigneeAvatar = ReportUtils.getAvatar(lodashGet(props.personalDetails, [props.report.managerEmail, 'avatar']), props.report.managerEmail);
```

Comment:
I tried to remove the assignee and just replace it with manager because we use manager as the assignee in all our other flows. It should be the same thing but I might have missed a few things